storage-plans.yaml

```diff
@@ -186,4 +186,4 @@ aurora-pg: {}
 
 s3: {}
 
-external-s3: {}
+s3-policy: {}
```

Comment:
This change cannot be merged till https://github.com/uktrade/copilot-tools/pull/21 is merged. Should this change be made in a subsequent PR or do we wait for the linked PR to merged first?